### ISSUE TITLE
TypeError: Cannot set property 2011-06-15 of #<.> which has only a getter

### ISSUE THREAD

**Issue by apdarshan** [amazon-connect/amazon-connect-streams#577]
Hello,

I am trying to use amazon-connect-streams sdk in my vue application. The library is throwing following error as soon as I import module using import 'amazon-connect-stream';

```
vue-router.esm-bundler.js:3325 TypeError: Cannot set property 2011-06-15 of #<Object> which has only a getter
    at 105.../apis/connect-2017-02-15.min (connect-streams.js:21051:33)
    at o (connect-streams.js:4:265)
    at r (connect-streams.js:4:431)
    at node_modules/amazon-connect-streams/release/connect-streams.js (connect-streams.js:4:459)
    at __require2 (chunk-7FP5O474.js?v=39c8a219:10:50)
    at dep:amazon-connect-streams:1:16
```

I am currently blocked on this issue, please help.

**Comment by stackasaur** [comment]
Duplicate issue of #565,

This issue is fixed in the following PR #574.

**Comment by martin-greenberg** [comment]
Closing as duplicate of #565, this issue should be fixed in Streams 2.3.0.
